common/tests/deployment_test.yaml

```diff
@@ -0,0 +1,61 @@
+suite: test deployment
+templates:
+  - deployment.yaml
+tests:
+  - it: should pass all kinds of assertion
+    values:
+      - ./values/global.yaml
+      - ./values/values.yaml
+      - ./values/image.yaml

```

Comment:
I take it from the assertion image.yaml trumps values.yaml here. Presumably the files are applied from bottom to top?

Reply:
Oh sorry I meant to say top to bottom :)